news/three-months-at-the-rust-foundation.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2023-07-01T12:00:00Z
+title: My first three months at the Rust Foundation
+description: Time flies when you're having fun… at the Rust Foundation
+byline: Tobias Bieniek, crates.io team co-lead and Rust Foundation software engineer
+tags:
+  - foundation
+layout: layouts/news.njk
+---
+
+In May 2022, I received an exciting email from the Rust Foundation:
+
+> […] you have been provisionally selected to be one of our first Rust Foundation Fellows
+
+Fast-forward one year, and that fellowship grant has transformed into a full-time role at the Rust Foundation. My primary focus now revolves around the maintenance and feature development for [crates.io](https://crates.io), the official Rust package registry.
+
+As I reflect on my first three months in this position, I want to take this opportunity to highlight some of the significant milestones I have achieved:
+
+## Improved API tokens
+
+One of the most noticeable achievements during this period has been the substantial improvements made to API tokens on crates.io.
+
+With the introduction of token scopes our users can now restrict their API tokens to specific crates and operations, for instance only allowing a token to publish new versions of a specific crate.
```

Comment:
> The second bit sounds repetitive but perhaps I'm overlooking some technical context.

it's the most common use-case for the token scopes, so I figured it would make sense to explicitly mention it.